backend/v3/domain/settings.go

```diff
@@ -0,0 +1,286 @@
+package domain
+
+import (
+	"context"
+	"encoding/json"
+	"net/url"
+	"time"
+
+	"github.com/zitadel/zitadel/backend/v3/storage/database"
+)
+
+//go:generate enumer -type SettingType -transform snake -trimprefix SettingType -sql
+type SettingType uint8
+
+const (
+	SettingTypeUnspecified SettingType = iota
+	SettingTypeLogin
+	SettingTypeLabel
+	SettingTypePasswordComplexity
+	SettingTypePasswordExpiry
+	SettingTypeDomain
+	SettingTypeLockout
+	SettingTypeSecurity
+	SettingTypeOrganization
+)
+
+type Setting struct {
+	ID         string          `json:"id,omitempty" db:"id"`
+	InstanceID string          `json:"instanceId,omitempty" db:"instance_id"`
+	OrgID      *string         `json:"orgId,omitempty" db:"org_id"`
+	Type       SettingType     `json:"type,omitempty" db:"type"`
+	LabelState *LabelState     `json:"labelState,omitempty" db:"label_state"`
+	Settings   json.RawMessage `json:"settings,omitempty" db:"settings"`
```

Comment:
the domain is not interested in the raw message so i would remove it. if we need it in storage i would create a custom struct in the repository package